src/Plugin/Field/FieldType/AutomatedListingConfiguration.php

```diff
@@ -68,8 +68,10 @@ protected static function getContainer() {
    */
   public static function defaultStorageSettings() {
     return [
-      'index' => 'node',
-    ] + parent::defaultStorageSettings();
+        'index' => 'node',
+        'expose_filter_operator' => 0,
```

Comment:
hi @asarfara cc @anthony-malkoun,

These settings should be in field widget settings, not field storage settings.

Reply:
Yep will update it :)